docs/permissions.md

```diff
@@ -0,0 +1,151 @@
+# Permissions
+
+`marsha` user access and permissions are managed following two different approaches:
+
+- For the LTI part, the authentication and authorization are managed by the LTI consumer site.
+  The LTI consumer site is responsible for authenticating the user and determining
+  their `role`. The user ID and Role are then passed to `marsha` through the LTI
+  request.
+- For the standalone site, the authentication and authorization are managed by `marsha`.
+  `marsha` can also rely on external SSO (see [renater_federation_saml.md](renater_federation_saml.md)).
+
+
+## LTI
+
+### Authentication
+
+The user is authenticated by the LTI consumer site, and `marsha` relies on the data provided by the signed LTI request.
+Once the first LTI request has been received, `marsha` will provide a pair of access/refresh JWT for the frontend 
+to authenticate requests.
+These JWT contains the authentication/authorization information provided by the original LTI request.
+
+### Authorization
+
+The user's `role` is provided by the LTI consumer site, and `marsha` relies on the data provided by the signed LTI request.
+
+
+## Standalone site
+
+### Authentication
+
+The user is authenticated by `marsha` and can be either:
+
+- an "internal" Django user (_i.e._ a user created and managed by `marsha` itself, 
+  but for now there is no registration process for external users to use this).
+- a user authenticated by an external SSO (see [renater_federation_saml.md](renater_federation_saml.md))
+
+The frontend application provides the login page, the backend will handle the authentication process and 
+generate a pair of access/refresh JWT for the frontend to make requests.
+These JWT only contains the authentication information (user ID).
+
+### Authorization
+
+Authorizations are built around several models:
+
+- `OrganizationAccess` links a user to an organization and a role for this organization.
+- `ConsumerSiteAccess` links a user to a consumer site and a role for this consumer site.
+- `PlaylistAccess` links a user to a playlist and a role for this playlist.
+
+For each one, a user can only have one role (_.e.g._ only one link between a user and a playlist).
+
+#### Organization access
+
+A user is linked to an organization through the `OrganizationAccess` model. This can provide three roles:
+
+- `ADMIN`: the user is an administrator of the organization.
+- `INSTRUCTOR`: the user is an instructor of the organization.
+- `STUDENT`: the user is a student of the organization.
+
+##### Organization administrator
+
+The organization administrator should be able to do "everything" regarding the organization. 
+Among other things, they can: 
+
+- 🚧 Create and manage (read/write) all the organization's access control.
```

Comment:
What is the meaning of 🚧 ? A work in progress ?
Maybe we should add a note about it ?

Reply:
I've squashed the fixup, sorry, I added a note here : https://github.com/openfun/marsha/pull/1977/files#diff-8300d5dd12494e4952d9b3f5f7e3d9fa4c7e7ef65be7eafa37d7614716e851f0R51